common/coap-server/src/main/java/org/thingsboard/server/coapserver/TbCoapDtlsDeviceAddr.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright © 2016-2024 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.coapserver;
+
+import lombok.Data;
+import org.thingsboard.server.common.data.id.DeviceId;
+
+import java.net.InetSocketAddress;
+
+@Data
+public class TbCoapDtlsDeviceAddr {
```

Comment:
Let's rename to `TbCoapDtlsSessionKey`, as it is used for dtlsSessionsMap

Reply:
Ok